pkg/util/azureclient/azureclient.go

```diff
@@ -0,0 +1,75 @@
+package azureclient
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/marketplaceordering/mgmt/marketplaceordering"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	azmarketplaceordering "github.com/Azure/azure-sdk-for-go/services/marketplaceordering/mgmt/2015-06-01/marketplaceordering"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-02-01/storage"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
```

Comment:
We probably need to start a document with the conventions we are following in our codebase (@jim-minter). Re imports we use the following convention:
```go
import (
  "stdlib imports"
  // empty line
  "third party imports"
  // empty line
  "same repo (openshift-azure) imports"
)
```
If we want to start viewing import graphs we will need to use `go list` but this conventions makes the code more readable and our packages are small enough to allow some utility out of this (you can quickly figure out third part deps in a package w/o using `go` magic).